docs/reservas_canchas_udla_db.sql

```diff
@@ -52,7 +54,7 @@ CREATE TABLE Reserva (
     Hora_fin TIME NOT NULL,
     Estado VARCHAR(10) NOT NULL,
     CONSTRAINT solicitudFK FOREIGN KEY (SolicitudID) REFERENCES Solicitud(ID),
-	CONSTRAINT canchaFK FOREIGN KEY (CanchaID) REFERENCES Cancha(ID),
+	CONSTRAINT canchaFK2 FOREIGN KEY (CanchaID) REFERENCES Cancha(ID),
```

Comment:
En caso de repertirse el nombre un FK busquemos una alternativa que mantenga la legibilidad, si ponemos el 2 al final podemos terminar con muchos iguales y no sabremos diferenciar
```suggestion
	CONSTRAINT solictud_canchaFK FOREIGN KEY (CanchaID) REFERENCES Cancha(ID),
```